config/checkstyle/suppressions.xml

```diff
@@ -6,4 +6,5 @@
 
 <suppressions>
     <suppress checks="[a-zA-Z0-9]*" files="[\\/]generated[\\/]" />
+    <suppress checks="[a-zA-Z0-9]*" files="CustomTitledPaneSkin.java" />
```

Comment:
Could you add a comment why the file is completely excluded? Can't the checkstyle issues be fixed? (https://medium.com/@matryer/broken-windows-theory-why-code-quality-and-simplistic-design-are-non-negotiable-e37f8ce23dab ^^)

Reply:
Added a comment because it's a modified variant of the original Skin.